Resources/Catalog/FileCatalogProxyClient.py

```diff
@@ -31,6 +32,12 @@ def getName( self ):
     """
     return self.fcName
 
+  def setInterfaceMethods( self, methodTuple ):
+    self.interfaceMethods = methodTuple
+
+  def getInterfaceMethods( self ):
+    return self.interfaceMethods
+
```

Comment:
I see. Probably in this case, to be more python, one should use properties, but I am not sure by Catalog object construction it would work.